src/app/shared/top-panel/user-panel/invite-user/invite-user-dialog/new-user/new-user.component.html

```diff
@@ -29,20 +29,20 @@
            [placeholder]="emailPlaceHolder()"
            (keypress)="onKeyPress($event)"
            (input)="onInputChanged(emailField.control.value)" email>
-    <small id="emailValid" class="invalid-message" *ngIf="emailField.invalid" i18n="@@inviteUser.dialog.user.email.invalid">
-      Please enter a valid e-mail address (e.g. alan.turing@lumeerio.com).
-    </small>
-    <small class="invalid-message" *ngIf="isDuplicate" i18n="@@inviteUser.dialog.email.duplicate">
-      A colleague with this email is already invited. Try another!
-    </small>
-
     <button type="button"
             class="btn btn-sm btn-primary"
             [disabled]="emailField.invalid || !email || isDuplicate"
             (click)="onAddUser()">
       <i class="fas fa-plus mr-2"></i>
       <span class="font-weight-bold" i18n="@@inviteUser.dialog.button">Add</span>
     </button>
+    <small id="emailValid" class="invalid-message" *ngIf="emailField.invalid && emailField.touched" i18n="@@inviteUser.dialog.user.email.invalid">
+      Please enter a valid e-mail address (e.g. alan.turing@lumeerio.com).
+    </small>
+    <small class="invalid-message" *ngIf="isDuplicate" i18n="@@inviteUser.dialog.email.duplicate">
+      A colleague with this email is already invited. Try another!
```

Comment:
I think `Try another one!` feels more natural.

Reply:
True! Thank you for the suggestion. I will fix that with another PR..